manifests/install.pp

```diff
@@ -4,6 +4,11 @@
 #
 class confluence::install {
 
+  File {
```

Comment:
This is very dangerous as it might pollute other file resources as well. can you revert this?

Reply:
I can... but it is also used in the class confluence::config.... so there is no difference?!

Reply:
yes, but it would be good to not introduce it in more places.